pypeit/spectrographs/keck_kcwi.py

```diff
@@ -860,7 +860,7 @@ def get_detector_par(self, det, hdu=None):
             # Some properties of the image
             binning = self.compound_meta(self.get_headarr(hdu), "binning")
             numamps = hdu[0].header['NVIDINP']
-            specflip = True if hdu[0].header['AMPID1'] == 2 else False
+            specflip = False if hdu[0].header['AMPMODE'] == 'ALL' else True
```

Comment:
I would comment on this in the change log.  Is this a general change for all KCWI taken, or does it reflect a change in header that needs an associated date `if..else` check?